mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -77,8 +77,14 @@ CLASS net/minecraft/unmapped/C_sddaxwyk net/minecraft/item/ItemStack
 			COMMENT the count of items
 	METHOD m_bqtrkfmc useOnBlock (Lnet/minecraft/unmapped/C_yyklzime;)Lnet/minecraft/unmapped/C_ozuepbyj;
 		ARG 1 context
+	METHOD m_caefvdpe damageEquipment (ILnet/minecraft/unmapped/C_gmbqjnle;Lnet/minecraft/unmapped/C_usxaxydn;Lnet/minecraft/unmapped/C_yuycoehb;)Lnet/minecraft/unmapped/C_sddaxwyk;
 	METHOD m_cgmsskmm split (I)Lnet/minecraft/unmapped/C_sddaxwyk;
 		ARG 1 amount
+	METHOD m_cgmunnul copyComponentsToNewStack (Lnet/minecraft/unmapped/C_gmbqjnle;)Lnet/minecraft/unmapped/C_sddaxwyk;
+		COMMENT {@return a new item stack with the components and stack size copied from this stack, unless this stack is empty, then returns an empty item stack}
+		COMMENT
+		COMMENT @see #copyComponentsToNewStack
+		COMMENT the resulting item stack's item
```

Comment:
seems like you've fallen victim to an enigma bug here. while i fix this, please right-click and use 'mark as deobf' on gray parameters before adding javadoc. i'm going to mark every spot on this PR where it needs fixing, really sorry that we didn't catch this issue sooner.

Reply:
to clarify, the issue is that the javadoc isn't associated with the parameter since the name isn't saved (gray highlight means it's been automatically mapped and doesn't need to be saved in the mapping files). since we don't save the name, the javadoc doesn't get properly attached to the parameter.